pkg/logger/logger.go

```diff
@@ -18,26 +21,65 @@ const (
 	LogLevelDebug   LogLevel = "Debug"
 	LogLevelInfo    LogLevel = "Info"
 	LogLevelWarning LogLevel = "Warning"
+	LogLevelError   LogLevel = "Error"
 )
 
-// logLevelOrder defines the order of log levels from most verbose to least verbose
+// AtmosTraceLevel is a custom log level one level lower than DebugLevel for more verbose logging.
+const AtmosTraceLevel log.Level = log.DebugLevel - 1
+
+// logLevelOrder defines the order of log levels from most verbose to least verbose.
 var logLevelOrder = map[LogLevel]int{
 	LogLevelTrace:   0,
 	LogLevelDebug:   1,
 	LogLevelInfo:    2,
 	LogLevelWarning: 3,
-	LogLevelOff:     4,
+	LogLevelError:   4,
+	LogLevelOff:     5,
 }
 
 type Logger struct {
-	LogLevel LogLevel
-	File     string
+	LogLevel    LogLevel
+	File        string
+	AtmosLogger *log.Logger
 }
```

Comment:
This seems odd to me. Isn't AtmosLogger our own custom struct as well? Why do we need nested structs in this situation?